$NetBSD: patch-libqpdf.pc.in,v 1.2 2023/04/24 07:38:38 wiz Exp $

graphics/jpeg does not provide any `.pc' files and adding it to the list of
`Requires.private:' breaks all packages that need to retrieve pkg-config
information for libqpdf.

--- libqpdf.pc.in.orig	2023-02-25 21:24:41.000000000 +0000
+++ libqpdf.pc.in
@@ -6,6 +6,6 @@ includedir=@CMAKE_INSTALL_FULL_INCLUDEDI
 Name: libqpdf
 Description: PDF transformation library
 Version: @PROJECT_VERSION@
-Requires.private: zlib, libjpeg@CRYPTO_PKG@
+Requires.private: zlib
 Libs: -L${libdir} -lqpdf
 Cflags: -I${includedir}
